x-pack/plugins/security_solution/server/lib/detection_engine/signals/preview/preview_rule_execution_log_client.ts

```diff
@@ -64,6 +68,11 @@ export const createWarningsAndErrors = () => {
       warningsAndErrorsStore.push(args);
       return Promise.resolve();
     },
+
+    clearWarningsAndErrorsStore() {
+      // Clears the array
```

Comment:
I don't think we need the comment given the name of the function. that's neat though I didn't know you could clear an array like that!